components/channel/internal/channelhealth/channelHealth.go

```diff
@@ -0,0 +1,73 @@
+package channelhealth
```

Comment:
It doesn't seem to be idiomatic go to use names like channelHealth.go usually multiple words in go file names are separated by underscores such as channel_health.go

Reply:
i agree and would just call it "health" ?

Reply:
I changed the channel and dispatcher to simply be "health" (which is the same as the common package, but I think that only means the import needs an override prefix).  Seems to work.